platform-sdk/platform-apps/tests/PlatformTestingTool/src/main/java/com/swirlds/demo/platform/PlatformTestingToolMain.java

```diff
@@ -711,12 +722,16 @@ private void initBasedOnPayloadCfgSimple(final PayloadCfgSimple pConfig) {
     }
 
     private void initializeAppClient(final String[] pars, final ObjectMapper objectMapper) throws IOException {
```

Comment:
In `swirlds-platform-regression` repo I couldn't find any test config, which sets more than one param to PTT. It means, this method doesn't seem to be called at all. Still, I decided to leave it as is and use parameter at index 2 for "use MerkleDb" flag